pkg/daemon/list.go

```diff
@@ -20,7 +20,7 @@ func (srv *Daemon) listHandler(engine api.Engine) func(w http.ResponseWriter, r
 		plans := engine.TestCensus().ListPlans()
 		for _, tp := range plans {
 			for _, tc := range tp.TestCases {
-				_, err := tgw.Write([]byte(tp.Name + "/" + tc.Name + "\n"))
+				_, err := tgw.WriteProgress([]byte(tp.Name + "/" + tc.Name + "\n"))
```

Comment:
All of this is technically part of the result, not progress. Ideally we'd create a `strings.Builder`, write to it, then write the full string as the Result via `WriteResult`.

Reply:
I see what you mean. Technically we were using `Write` previously, which was directly going to the `progressWriter`, so here we keep the same semantics.

How we define `progress` and `result` in the `list` cmd is up to us, so you could say that all of it is the Result, or individual Progress messages.